packages/storybook/stories/components/input/3_demo.stories.ts

```diff
@@ -139,3 +130,65 @@ type ValidationProps = {
   ...(extractStoryParams(storyParams) as Record<string, unknown>),
 };
 (Validation as unknown as ValidationProps).play = InputPlay;
+
+/* All Inputs */
```

Comment:
This part has been removed in Storybook refactor step 3. 
Please remove "All Inputs" example here since we defined that only components like Flag and Icon should keep an "All" story (usage is described in Usage Guidelines)